docs/markdown/Getting Started/getting-started/existing-repositories.md

```diff
@@ -31,6 +31,21 @@ First, run [`./pants tailor ::`](doc:initial-configuration#5-generate-build-file
 
 Then, activate the [Linters and formatters](doc:python-linters-and-formatters) you'd like to use. Hook up the `fmt` and `lint` goals to your [CI](doc:using-pants-in-ci).
 
+If you're not ready to run a particular linter or formatter on certain parts of your project yet, you can combine [`__defaults__`](doc:targets#field-default-values) with the `skip_x` fields, e.g.:
+
+```python src/python/project/BUILD
+__defaults__(
```

Comment:
I have to say that the double underscore feels weird to me.
as a python eng, I can kind of infer what it means (and it makes it feel like something advanced or internal/private that I should not be using) and I think for non-python engineers this will also feels like that since underscore prefixes usually mean private and or internal' stuff and not part of the API users should be using (as opposed to other stuff in a BUILD file, like python_sources, python_tests, etc...)

Reply:
@kaos @thejcannon @benjyw @stuhood thoughts? It's not too late to change to `defaults()`, which is nice that it's easier to type also. 

I think we wanted to avoid confusion if people thought it was a target, right? Would it be weird to do `set_defaults()` to avoid that confusion? Targets are nouns, whereas this is a verb.

Reply:
Asher has a point, it is a bit weird and out of step with any other BUILD file convention we have. 

`set_defaults()` seems fine to me.

Reply:
Yeah, thats a good point. Damn my Python-tinted goggles.

I _kind of_ still want a way to discern it from normal targets. `DEFAULTS(...)`? Otherwise if thats too knife-in-the-eye `set_defaults` is good with me.

I wish we could `help` it and have it on the docsite :/

Reply:
I like that idea of have `__defaults__` show up in help.

Reply:
> as a python eng, I can kind of infer what it means (and it makes it feel like something advanced or internal/private that I should not be using)

This doesn't really add up for me. In python `__foo` is private, where as `__foo__` is a language construct, such as for the commonly used `__init__` or less frequently but in no way private or truly "internal" `__getitem__` or `__lt__` etc.

Here, the idea with `__defaults__` is to signal that is a special thing (from a C background, I'd call it a pragma instruction), not just another target declaration. And while still having it be a function call, we're a bit limited in what syntax to pick, unless we want to introduce additional passes over the BUILD files (to parse comments or some such).

Reply:
Oh, right. This has been addressed and closed already in #16394